.github/workflows/job-precommit-python-runners-portability-flink.yml

```diff
@@ -0,0 +1,89 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Test for Python Precommit Runners Portability Flink
+
+
+name: Python Precommit Runners Portability Flink
+on:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+    paths: [ 'sdks/python/apache_beam/runners/portability/flink_runner.py',
```

Comment:
We should run postcommits on all paths, please remove these

Reply:
Same [context](https://github.com/apache/beam/pull/23364#discussion_r999969683) @damccorm

Reply:
is there any comment?